### DISCUSSION TITLE
How to enforce revisions policy by API

### DISCUSSION THREAD

**Discussion by bwmeyer** [ravendb/ravendb#17304] [Q&A]
How can I enforce the revisions policy by API?

I have updated the policy by sending a ConfigureRevisionsOperation to the server.
The policy will be overwritten correctly, but I don't see an option to enforce the policy via API.
Currently I need to go to the Studio and click on the "Enforce Configuration"  button.

**Comment by ayende** [comment]
```
var result = await store.Operations.SendAsync(new EnforceRevisionsConfigurationOperation());
var operationResult = (EnforceConfigurationResult)await result.WaitForCompletionAsync();
```

**Comment by bwmeyer** [comment]
Sounds promising, but unfortunately the class **EnforceRevisionsConfigurationOperation** is unknown by the system.
Am I missing something?
I cannot find the source for EnforceRevisionsConfigurationOperation on GitHub in the current branch nor for 6.0.

**Reply by ayende** [reply]
It's here: https://github.com/ravendb/ravendb/blob/v6.0/src/Raven.Client/Documents/Operations/Revisions/EnforceRevisionsConfigurationOperation.cs#L12

**Reply by bwmeyer** [reply]
I can see it is available for v6.0 (even GitHub search was not working)
But it seems for 5.4 it is not available:

<img width="817" alt="image" src="https://github.com/ravendb/ravendb/assets/39671001/7520f25a-ec92-4342-829c-a6a11e224cd7">

**Reply by ppekrol** [reply]
Hi, it was added in 6.0. You can copy the operation to your application that uses 5.4 Client API and use it there.

**Reply by bwmeyer** [reply]
Thank you for the clarification.
